homeassistant/components/husqvarna_automower/services.yaml

```diff
@@ -0,0 +1,31 @@
+start_for:
+  target:
+    entity:
+      integration: "husqvarna_automower"
+      domain: "lawn_mower"
+  fields:
+    duration:
+      required: true
+      example: 60
+      default: 100
+      selector:
+        number:
+          min: 1
+          max: 60480
+          unit_of_measurement: "min"
```

Comment:
A maximum of 60480 minutes is a bit ridiculous, it's 42 days.
Why not a duration selector?

Reply:
I didn't know, there is a duration selector. Added in https://github.com/home-assistant/core/pull/117269/commits/de54a4e4dd49dea6a2745f8c20f0415bf8ab048a